api/src/main/java/org/openmrs/obs/handler/AbstractHandler.java

```diff
@@ -78,10 +78,14 @@ public File getOutputFileToWrite(Obs obs) throws IOException {
 			outputfile = new File(dir, title + "." + extension);
 			// outputfile = new File(dir, title);
 		}
-		
+		// Get MIME type
+		String mimetype = "";
+		if (StringUtils.isNotBlank(obs.getComplexData().getMimeType())) {
+			mimetype = "_" + obs.getComplexData().getMimeType().replace("/", "-slash-");
```

Comment:
It doesn't matter since this slash is for parsing MIME type and filename in both UNIX and Windows will look like "`somefile_image-slash-jpg`".